### DISCUSSION TITLE
New price plan

### DISCUSSION THREAD

**Discussion by JCV67** [Budibase/budibase#12333] [General]
Today is  a sad day for selfhost.

**Comment by Jednadvacet** [comment]
Agreed.

**Comment by bmscmoreira** [comment]
I believe the change is the 20 user limit, where before there was no limit?

**Comment by AndrewTardif** [comment]
RIP, one of the major reasons why I chose Budi over others when doing research for the right tool was because of the unlimited users for self hosting. I guess I can still keep the app on an old version though.

**Comment by bmscmoreira** [comment]
Well, yes, I am very new to BB. One of the reasons I liked it was because of the selfhosting freedom, now this makes me a bit anxious about the future.

**Comment by SaschaValentin** [comment]
I absolutely agree to you all. I´ve just read the plans in limiting the amount of users for self-hosting.

Totally agree when a company wants to get a fair wage for what they produce. And to be sure: BB is a very powerful and handy application.

But, unfortunately I think the team assume that BB is only used in business environment. They disregard that it could also be used in other ways. In my case for example I´ve used BB to build a website that is used by the citizens of my village. Okay, one could argue why I´m not using wordpress for something like this and instead using a software that´s not made for this. But I can counter that BB has all the functionality out of the box where I would need a huge amount of plugins to realize in wordpress.

And as you can see: I really can´t control the number of users and even less I could not pay for it, because I don´t get any money for this. I´ve just build this app to give the people a chance to stay in touch to each other. Assuming there are 1000 users and every user would be charged with 10 Euro - how should I pay 10000 Euro a month?? Sure, you can visit this website without registering, but I have also some functionality on the site that requires the user to be logged (for example a small chat). 

I might be an isolated case but I don´t think so. Other people will come up with ideas like this too. They do not take such usage scenarios into account in their considerations. Unfortunately, this makes BB uninteresting for such uses in the future, even though it is such a great platform.

I would be very happy if they would reconsider this decision and instead implement something that continues to enable such use. This could be one user role with limited privileges (only to conditionally show/hide things on the frontend) that is excluded from the count of users. Or the possibility of receiving at least critical security updates for new versions if you decide to continue using an old version in which the user limit does not yet exist. So that you don´t get new features but it is safe to use BB.

I hope those responsible can think about such solutions again.

**Reply by mjashanks** [reply]
Hi @SaschaValentin - the team has already replied to you over email, but just wanted to share the response here for visibility.

The good news is that you can stay on the current version of Budibase, which will never be affected by the limits. This means that you won't receive future updates - but there should be no urgency on your side right now.

If you do require updates, we offer charity discounts and volume discounts. If you would like a paid contract, we're happy to have a chat just get in touch via sales@budibase.com

I understand that the limits are difficult for use cases like this. As everyone has pointed out, we are a business and we do need to make money to continue developing the product - this was a necessary step for us. We will try to be as accommodating as possible for non-profit use cases.

What this change will mean is that we will be able to introduce **more** features into the open-source version, as we a re no longer as concerned with using features to drive monetization. 

We'd love to hear from you (or anyone else in this thread) regardless - please do reach out and chat to us on a call.

**Comment by helioguardabaxo** [comment]
Hi, @mjashanks !
What is latest version without the limitations for self hosted installations?

**Reply by manwithpowers** [reply]
The latest version that I'm able to pull without user limits is 2.13.15

**Reply by helioguardabaxo** [reply]
Thanks!

**Reply by gmentzik** [reply]
2.13.17 but unfortunatelly has a bug in component conditions at more than , less than number comparison. :( That bug existed since 2.13.14.

**Comment by insomniacj93** [comment]
> RIP, one of the major reasons why I chose Budi over others when doing research for the right tool was because of the unlimited users for self hosting. I guess I can still keep the app on an old version though.

Hi everyone, i wouldn't mind either keeping it on the old release before the change was applied, to this day i still havent had more than a couple persons using the app but if in the future i get 21 i would be forced to pay a price that greatly exceeds my outcome (i get paid minimum wage and this is a company project)... I love budibase, it is the strongest web based app developement application i've ever used. I understand the fact they need to monetize this things, I'm just sad that what once was free, slowly starts to be paid..

With this said, is there any chance that anyone knows which release happened before the change and how to properly update budibase into old releases?

I'm using docker-compose and I'm currently on 2.11

Best regards to the community and the staff!

**Reply by manwithpowers** [reply]
The latest single image that works is 2.13.15.
In your docker-compose.yml you should be referencing
`image: budibase/budibase:2.13.15`

**Comment by Kakovisk** [comment]
Can we continue the open source code for hosted versions? From version 2.13.15?

**Comment by thanhpd56** [comment]
Hello @all, Can I folk Budibase repo then update the code to remove this limitation?

**Reply by Kakovisk** [reply]
I believe so, as it is an open-source project under the GPL v3 license. This means we have the freedom to:

Download and run the code locally.
Modify the code to meet our needs.
For distribution, there are legal requirements that must be followed.

Could anyone else share their opinion?

**Reply by gfxguru** [reply]
I'm totally in agreement with you on this. This is the first move away from open source in my opinion. Glad I did some research on the platform before diving in.

**Comment by dbaccello** [comment]
Current plan on https://budibase.com/pricing/ still saying "Unlimited users" for Self-Host.

**Reply by manwithpowers** [reply]
This is actually new. It looks like the pricing has changed sometime in the past few weeks: https://web.archive.org/web/20260102021202/https://budibase.com/pricing/

**Reply by PClmnt** [reply]
Yea, we've updated it :)

**Reply by manwithpowers** [reply]
@PClmnt It looks like this isn't yet reflected in the self-hosted app (just updated to the latest version) or on the tenancy management page at account.budibase.app. Any idea when that will happen?
<img width="643" height="362" alt="Screenshot 2026-01-22 at 11 34 19" src="https://github.com/user-attachments/assets/5eebed13-38ce-4fd9-8e24-d095cf223d54" />
<img width="993" height="833" alt="Screenshot 2026-01-22 at 11 30 23" src="https://github.com/user-attachments/assets/aa8924d4-38d7-49a9-bb50-5a70e9831b4d" />

**Reply by jmlaurenti** [reply]
Hi @PClmnt , same remark as @manwithpowers with self hosted instance, even with very latest version (3.25.4), still see the 20 users limit..

And thanks for the hard work ! :-)

**Reply by mjashanks** [reply]
We'll take a look at this...

**Reply by mjashanks** [reply]
Should be sorted now!

**Reply by manwithpowers** [reply]
Unfortunately, I'm still limited to 20 users after updating to version 3.26.1
<img width="408" height="311" alt="Screenshot 2026-01-28 at 19 45 14" src="https://github.com/user-attachments/assets/ea05f61e-e264-45a1-ae41-c10369097c50" />
<img width="784" height="372" alt="Screenshot 2026-01-28 at 19 45 06" src="https://github.com/user-attachments/assets/7cb9e2ca-86cd-4850-aa6b-d2b7daf86682" />
<img width="403" height="254" alt="Screenshot 2026-01-28 at 19 44 39" src="https://github.com/user-attachments/assets/9fb88361-6389-44b8-bd12-c0cc5e5f6e9d" />

**Reply by NDCallahan** [reply]
@manwithpowers - I just checked mine I'm unlimited now on the free plan.

**Reply by dbaccello** [reply]
me too

**Reply by jmlaurenti** [reply]
All good for me too ! Thanks !

**Reply by manwithpowers** [reply]
👍 I'm seeing the unlimited users after updating to 3.27.2 as well
